docs/trex_examples.md

```diff
@@ -24,12 +68,35 @@ Every Tableau extension has a manifest file (`.trex`) that describes the extensi
 
 To use the dashboard extension samples, you need to start up a web server on your computer to host the HTML pages. If you downloaded or cloned the Extensions API repository, you can start the web service in the root directory of the repository on your computer. 
 
-- From the root of this directory, start an HTTP server on port 8765.
-	- Python 2.x : `python -m SimpleHTTPServer 8765`
-	- Python 3.x : `python -m http.server 8765`
-	- Node.js : `npm install http-server -g && http-server -p 8765`
+1. Go to to the the `extensions-api` folder.
+2. To install the web server components, run the following command:
+   ```
+   npm install
+   ```
+3. To start the web server, run the following command:
+   ```
+   npm start
+   ```
+
+
+
+
+### Instructions for starting a web server on a different port
+
+The samples are set up so that the web server is using port `8765`.  If you need to specify a different port instead of `8765`, you can change ports using Python, if you have Python installed, or you can run another Node.js command.  
+
+From the `extensions-api` folder, start an HTTP server on port \<PORT\>, using one of the following commands:
```

Comment:
For the port section, maybe put it in the inline code thing like `PORT`

Reply:
Good idea. Someone might think they need to include the angle brackets.